contacts/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class NewContactForm(forms.Form):
```

Comment:
Rename this as `AddContactForm`. I believe that is more descriptive of its functionality.